app/controllers/system/admin/get_help_controller.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+class System::Admin::GetHelpController < System::Admin::Controller
+  def index
+    start_date = params[:start_at].presence
+    end_date = params[:end_at].presence
+
+    unless valid_date_range?(start_date, end_date)
+      return render json: { error: 'Invalid date range' }, status: :bad_request
+    end
+
+    fetch_all_recent_live_feedbacks(start_date, end_date)
+    user_ids = @get_help_data.map(&:submission_creator_id).uniq
+    assessment_ids = @get_help_data.map(&:assessment_id).uniq
+
+    load_assessment_question_hash(assessment_ids)
+    load_course_user_hash(user_ids)
+  end
+
+  private
+
+  def valid_date_range?(start_date_str, end_date_str)
+    return true unless start_date_str.present? && end_date_str.present?
+
+    start_date, end_date = parse_dates(start_date_str, end_date_str)
+    return false unless start_date && end_date
+    return false if start_date > end_date || (end_date - start_date).to_i > 365
+
+    true
+  end
+
+  def parse_dates(start_str, end_str)
+    [Date.parse(start_str), Date.parse(end_str)]
+  rescue ArgumentError
+    [nil, nil]
+  end
```

Comment:
nit: imo, the `parse_dates` function is not necessary, and adds additional contextual bloat to this controller.

```rb
  def valid_date_range?(start_date_str, end_date_str)
    return true unless start_date_str.present? && end_date_str.present?

    start_date, end_date = [Date.parse(start_date_str), Date.parse(end_date_str)]
    start_date <= end_date && (end_date - start_date).to_i <= 365
  rescue ArgumentError
    false
  end
```